content/en/tagging/_index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Getting started with tags
 kind: documentation
+description: "Tags add new dimensions to metrics"
```

Comment:
```suggestion
description: "Learn how to assign and use Tags with Datadog."
```

Reply:
thanks! I like your suggestion. :)

Reply:
Hi,

I don't have permission to merge. I got "code owner review required" message.

Thanks,